examples/kubernetes/README.md

```diff
@@ -0,0 +1,146 @@
+# :dango: Pipeline Orchestration on Kubernetes
+
+This example will demonstrate how to orchestrate pipelines using the ZenML Kubernetes-native orchestrator. We will build a simple pipeline consisting of four steps and orchestrate it in an Amazon EKS Kubernetes cluster.
```

Comment:
It would make sense here to explain the kubernetes orchestrator a bit more , its architecture, and why would people use it instead of airflow or AWS

Reply:
Please also say that in principal you can do it with any cloud backend  not just AWS

Reply:
In general this entire README is very specific to AWS, it would make sense to highlight in multiple places, that this is one of many ways.

Reply:
> Please also say that in principal you can do it with any cloud backend not just AWS

Good point, changed the description accordingly.

Reply:
> It would make sense here to explain the kubernetes orchestrator a bit more , its architecture, and why would people use it instead of airflow or AWS

Added short description and differences to other orchestrators at the start of the README.